packages/config/config/devices/0x000c/hs-wx300.json

```diff
@@ -0,0 +1,633 @@
+{
+	"revision": 1,
+	"manufacturer": "HomeSeer Technologies",
+	"manufacturerId": "0x000c",
+	"label": "HS-WX300",
+	"description": "Wall Dimmer & Switch",
+	"devices": [
+		{
+			"productType": "0x4447",
+			"productId": "0x4037"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"description": "Lifeline association only supports the \"manual reset\" event.",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Companion Sync",
+			"description": "Any device added to group 2 will be controlled. This allows other WX300 devices to used as companion switches. LEDS will be kept in sync.",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "3",
```

Comment:
A couple of these parameters are already templated, see other HomeSeer files for reference, e.g.
https://github.com/zwave-js/node-zwave-js/blob/master/packages/config/config/devices/0x000c/hs-ws200.json

We should do the same here.

Reply:
I've flipped everything I could find that matches to templates, as well as added a new homeseer template stanza for individual LED blink status (led_blink_status) since it was repeated 7 times in this device-config